introduction_to_applying_machine_learning/visual_object_detection/src/xml2json.py

```diff
@@ -0,0 +1,178 @@
+# Use this script to convert annotation xmls to a single annotations.json file that will be taken by Jumpstart OD model
+# Reference: XML2JSON.py https://linuxtut.com/en/e391e5e6924945b8a852/
+
+import random
+import subprocess
+subprocess.run("python -m pip install opencv-python-headless xmltodict", shell=True)
```

Comment:
I don't really like to see random installations happening behind the curtain... should be discussed in a prerequisites section of your readme / notebook.

Reply:
Thanks, I removed all the behind-the-scene installation in the python files, and install them at the beginning of the notebook.